config/database.yml

```diff
@@ -8,8 +8,8 @@ default: &default
   adapter: postgresql
   pool: 5
   timeout: 5000
-  username: root
-  password: root
+  username: zstayman
```

Comment:
Probably we don't want to commit this. But then, `root` doesn't work for me, either, so I don't mind.